com.github.VescTool.yaml

```diff
@@ -0,0 +1,19 @@
+app-id: com.github.VescTool
+runtime: org.kde.Platform
+runtime-version: '5.13'
```

Comment:
This is old and unsupported version. use 5.15 instead.

Reply:
I can check, but last release is using 5.13

I prefer @vedderb to decide on that.

Reply:
Where do you see that? 5.15 is explicitly mentioned in [build files](https://github.com/vedderb/vesc_tool/blob/4efde08566918c511e7b5d9c006b6b48160de672/build_lin#L63).

Also you should add specific commit to build from otherwise this won't be reproducible.

Reply:
Ouch!. Done